torchci/pages/metrics.tsx

```diff
@@ -560,72 +507,83 @@ export default function Page() {
+          <MasterCommitRedPanel params={timeParams} />
         </Grid>
-        <Grid container item xs={2} justifyContent={"stretch"}>
-          <Stack justifyContent={"space-between"} flexGrow={1}>
+
+        <Grid container item lg={2} md={3} xs={6} justifyContent={"stretch"}>
+          <Stack
+            justifyContent={"space-between"}
+            flexGrow={1}
+            flexWrap="wrap"
+            spacing={1}
+          >
             <ScalarPanelWithValue
               title={"% commits red on main (broken trunk)"}
               value={brokenTrunkRed}
               valueRenderer={(value) => (value * 100).toFixed(1) + "%"}
               badThreshold={(value) => value > 0.2}
             />
-            <ScalarPanel
-              title={"# commits"}
-              queryName={"num_commits_master"}
-              queryCollection={"commons"}
-              metricName={"num"}
-              valueRenderer={(value) => value}
-              queryParams={timeParams}
-              badThreshold={(_) => false}
-            />
-          </Stack>
-        </Grid>
-
-        <Grid container item xs={2} justifyContent={"stretch"}>
-          <Stack justifyContent={"space-between"} flexGrow={1}>
             <ScalarPanelWithValue
               title={"% commits red on main (flaky)"}
               value={flakyRed}
               valueRenderer={(value) => (value * 100).toFixed(1) + "%"}
               badThreshold={(value) => value > 0.2}
             />
-            <ScalarPanel
-              title={"# reverts"}
-              queryName={"reverts"}
-              metricName={"num"}
-              valueRenderer={(value: string) => value}
-              queryParams={timeParams}
-              badThreshold={(value) => value > 10}
-            />
           </Stack>
         </Grid>
 
-        <Grid container item xs={2} justifyContent={"stretch"}>
-          <Stack justifyContent={"space-between"} flexGrow={1}>
+        <Grid container item lg={2} md={3} xs={6} justifyContent={"stretch"}>
+          <Stack
+            justifyContent={"space-between"}
+            flexGrow={1}
+            flexWrap="wrap"
+            spacing={1}
+          >
             <ScalarPanel
               title={"% force merges due to failed PR checks"}
               queryName={"force_merge_red_avg"}
               metricName={"force_merges_red"}
               valueRenderer={(value) => (value * 100).toFixed(1) + "%"}
               queryParams={timeParams}
-              badThreshold={(value) => value > 0.2}
+              badThreshold={(value) => value > 0.055}
             />
-            <WorkflowDuration
-              percentileParam={percentileParam}
-              timeParams={timeParams}
-              workflowNames={["pull", "trunk"]}
+            <ScalarPanelWithValue
+              title={"% force merges due to impatience"}
+              value={"TBD"}
+              valueRenderer={(value) => value}
+              badThreshold={(value) => value > 0.055}
```

Comment:
To be frank, I would remove the placeholder, unless we can fill those in with some values right now

Reply:
I left the TBD queries in there to explain what would otherwise be gaps in the chart 2 of the three TBDs will be going away in a PR I’m just about to share. I expect/hope the third one to be addressed within a week